### DISCUSSION TITLE
Gates and Stiles

### DISCUSSION THREAD

**Discussion by pwb999** [organicmaps/organicmaps#2044] [Ideas]
I live in NW England and have used Organic Maps/Maps.Me for 3 years. 
Organic Maps is excellent as it shows the paths from OSM that are really useful for hiking/walking. However in this part of the whole there are many stone walls build by farmers to keep their sheep secure. The Gates and Stiles that cross these very sturdy structures are marked on OSM and although they are on the Organic Maps it is also impossible to see them. Could you consider marking these locations with a **S for Stile** and a **G for Gate.** This would be a very useful navigational aid.

Best wishes to the TEAM.

**Comment by biodranik** [comment]
Please provide us with screenshots and links to better understand the issue :) Foot navigation probably also should be improved by avoiding these objects.

**Comment by pastk** [comment]
Both gates and stiles are displayed in Organic Maps, do you use the latest app version and latest maps?

You might need to zoom in quite close to see them though and they're displayed as simple black dots.
There is an issue to use a better icon for these features https://github.com/organicmaps/organicmaps/issues/1919
